### PR TITLE
pass log level to KCM

### PR DESCRIPTION
**Pull request by atiratree**
needed for testing the KCM https://issues.redhat.com/browse/WRKLDS-604

It seems to me that log level is passed correctly to the other controllers

### CODE REVIEW

File: pkg/controllers/kube-controller-manager.go [hunk 1]
```diff
@@ -96,6 +96,8 @@ func configure(cfg *config.MicroshiftConfig) (args []string, applyFn func() erro
 	}
 
 	args, err = mergeAndConvertToArgs(overrides)
+	args = append(args, fmt.Sprintf("-v=%d", cfg.LogVLevel))
```
Comment (pacevedom) [inline pkg/controllers/kube-controller-manager.go:99]: Does including
```
	"v": {cfg.LogVLevel},
```
To `ExtendedArguments` in `overrides` do the same? You wouldnt need this line.

Comment (atiratree) [reply pkg/controllers/kube-controller-manager.go]: @pacevedom  Yup, looks better to use the same approach for both short and long arguments. I have modified `GetKubeControllerManagerArgs` to support that.

File: pkg/controllers/kube-controller-manager.go [hunk 2]
```diff
@@ -171,7 +173,11 @@ func GetKubeControllerManagerArgs(config map[string]interface{}) []string {
 	args := []string{}
 	for key, value := range extendedArguments.(map[string]interface{}) {
 		for _, arrayValue := range value.([]interface{}) {
-			args = append(args, fmt.Sprintf("--%s=%s", key, arrayValue.(string)))
+			if len(key) == 1 {
+				args = append(args, fmt.Sprintf("-%s=%s", key, arrayValue.(string)))
```
Comment (pacevedom) [inline pkg/controllers/kube-controller-manager.go:177]: This should work with [double dash](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-controller-manager/#options) too.

Comment (atiratree) [reply pkg/controllers/kube-controller-manager.go:177]: yes, you are right. But for example `-h` doesn't, so it shouldn't hurt anything to keep this here.

Comment (atiratree) [reply pkg/controllers/kube-controller-manager.go:177]: also a minor plus for consistency is that we are using `-v` in KCM operator as well when generating the args.

Nevertheless, I don't have a strong preference if you want to remove this.

Comment (oglok) [reply pkg/controllers/kube-controller-manager.go:177]: log level verbosity flag in KCM supports simple or double dash. So this sounds good to me.

**Pull request merged**